geode-core/src/test/java/org/apache/geode/security/ClientAuthorizationDUnitTest.java

```diff
@@ -53,7 +54,7 @@
  *
  * @since GemFire 5.5
  */
-@Category({DistributedTest.class, SecurityTest.class})
+@Category({DistributedTest.class, SecurityTest.class, FlakyTest.class}) // GEODE-3738
```

Comment:
do we want to add flaky to the entire class or just one particular test?

Reply:
The entire class. When there are failures, all the tests with current client and older server fail.